docs/api/tests.routing.generic.md

```diff
@@ -18,3 +20,15 @@ anta_title: ANTA catalog for generic routing tests
       filters:
         - "!test"
         - "!render"
+
+# Input models
+
+::: anta.input_models.routing.generic
+    options:
+      show_root_heading: false
+      show_root_toc_entry: false
+      show_bases: false
+      merge_init_into_class: false
+      anta_hide_test_module_description: true
+      show_labels: true
+      filters: ["!^**str**"]
```

Comment:
```suggestion

    options:
      show_root_heading: false
      show_root_toc_entry: false
      show_bases: false
      merge_init_into_class: false
      anta_hide_test_module_description: true
      show_labels: true
      filters: ["!^__str__"]
```

Reply:
@geetanjalimanegslab here toy need a newline to prevent markdownlint to misbehave